src/category_theory/limits/shapes/pullbacks.lean

```diff
@@ -605,6 +605,40 @@ lemma pushout.condition {X Y Z : C} {f : X ⟶ Y} {g : X ⟶ Z} [has_pushout f g
   f ≫ (pushout.inl : Y ⟶ pushout f g) = g ≫ pushout.inr :=
 pushout_cocone.condition _
 
+/--
+Given such a diagram, then there is a natural morphism `W ×ₛ X ⟶ Y ×ₜ Z`.
+
+  W  ⟶  Y
+    ↘      ↘
+      S  ⟶  T
+    ↗       ↗
+  X  ⟶  Z
+
+-/
+@[simp] def pullback_map {W X Y Z S T : C} (f₁ : W ⟶ S) (f₂ : X ⟶ S) [has_pullback f₁ f₂]
```

Comment:
Should this be called `pullback.map`? That seems in line with `pullback.lift` etc...
```suggestion
@[simp] def pullback.map {W X Y Z S T : C} (f₁ : W ⟶ S) (f₂ : X ⟶ S) [has_pullback f₁ f₂]
```

Reply:
Marking defs as `simp` is usually not what we want. It means that `simp` will always unfold this definition. But maybe that's what you actually intend?
Maybe you want an `abbreviation`?